types/vue-tel-input/index.d.ts

```diff
@@ -0,0 +1,94 @@
+// Type definitions for vue-tel-input 2.1
+// Project: https://educationlink.github.io/vue-tel-input
+// Definitions by: Komang Suryadana <https://github.com/suryadana>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.3
+
+import { VueConstructor } from 'vue';
+import { DirectiveOptions } from 'vue/types/options';
+
+export default VueTelInput;
+export const VueTelInput: VueTelInputConstructor;
+
+export interface VueTelInputProps {
+    value: string;
+    placeholder: string;
+    disabledFetchingCountry: boolean;
+    disabled: boolean;
+    disabledFormatting: boolean;
+    invalidMsg: string;
+    required: boolean;
+    defaultCountry: string;
+    enabledCountryCode: boolean;
+    enabledFlags: boolean;
+    preferredCountries: boolean;
+    onlyCountries: any[];
+    ignoredCountries: any[];
+    autocomplete: string;
+    name: string;
+    wrapperClasses: string;
+    inputClasses: string;
+    dropdownOptions: () => void;
+    inputOptions: () => void;
+    maxLen: number;
+}
+
+export interface VueTelInputData {
+    phone: string;
+    activeCountry: VueTelInputCountry;
+    open: boolean;
+    selectedIndex: any;
+    typeToFindInput: string;
+    typeToFindTimer: any;
+}
+
+export interface VueTelInputWatch {
+    state: (value: any) => void;
+    value: () => void;
+}
+
+export interface VueTelInputMethods {
+    initializeCountry: () => void;
+    getCountries: (list: any[]) => any[];
+    findCountry: (iso: string) => any[];
+    getItemClass: (index: number, iso2: string) => any;
+    choose: (country: VueTelInputCountry) => void;
+    onInput: ()  => void;
```

Comment:
Docs have this taking in an object argument.

Reply:
I have updated also the revision on [here](https://github.com/DefinitelyTyped/DefinitelyTyped/commit/98990b830958099c90df9b360415ceae34a388c3)